service/arm_templates/main.json

```diff
@@ -50,28 +50,93 @@
     "kubernetesIngressInsecurePort": {
       "type": "int"
     },
+    "masterCount": {
+      "type": "int",
+      "defaultValue": 1
+    },
+    "masterIPAddresses": {
+      "type": "array",
+      "metadata": {
+        "description": "An array containing all the master nodes' ip addresses."
+      }
+    },
+    "masterVMSize": {
+      "type": "string"
+    },
+    "masterDataDiskSizeGB": {
+      "type": "int"
+    },
+    "masterAdminUsername": {
+      "type": "string"
+    },
+    "masterAdminPassword": {
+      "type": "string"
+    },
+    "masterDisablePasswordAuth": {
+      "type": "bool",
+      "defaultValue": true
+    },
+    "masterAdminSSHKeyPath": {
+      "type": "string"
+    },
+    "masterAdminSSHKeyData": {
```

Comment:
Can we just use `masterAdminSSHKeyData` and remove `masterAdminSSHKeyPath`?

While we're developing I think it's good provide a master SSH key via the operator. It's also possible to create individual user accounts with SSH keys via `k8scloudconfig`. But that can come later.

https://github.com/giantswarm/clustertpr/blob/master/spec/kubernetes/ssh/user.go

Reply:
I tried removing it from the template, but it's not working and the cli throws an error telling the value of _linuxConfiguration.ssh.publicKeys.path_ is invalid.

I'll build it in the variable (/home/_masterAdminUsername_/.ssh/authorized_keys, that way the tpr won't need to provide it.